src/__tests__/codegenTypedDocuments.test.ts

```diff
@@ -138,4 +138,119 @@ describe("codegenTypedDocuments", () => {
       }"
     `);
   });
+
+  describe("module path customization", () => {
+    const queryDocument = parse(`
+      query authors {
+        authors {
+          idField
+        }
+      }
+    `);
+
+    const mutationDocument = parse(`
+      mutation createAuthor {
+        createAuthor {
+          idField
+        }
+      }
+    `);
+
+    const documents = [
+      { document: queryDocument, location: "literary/types/authors.gql" },
+      { document: mutationDocument, location: "mutations/createAuthor.gql" },
+    ];
+
+    const getPathConfig = (pluginConfig: Record<string, unknown>) => {
```

Comment:
Could you integrate this into the main `getConfig`, then we can use this also in case we add other options in the future.

e.g.

```ts
const getConfig = (
  generateOptions: Partial<Types.GenerateOptions> = {},
  pluginOptions: Partial<codegenTypedDocuments.UserConfig> = {}
): Types.GenerateOptions => ({
  filename: "not-relevant",
  schema: parse(printSchema(schema)),
  plugins: [
    {
      codegenTypedDocuments: {
        typesModule: "@codegen-types",
        ...pluginOptions,
      },
    },
  ],
  pluginMap: { codegenTypedDocuments },
  config: {},
  documents: [],
  ...generateOptions,
});
```

...

```ts
const config = getConfig({}, { modulePathPrefix: "stuff/" });
```